lib/ts/apidefs.h.in

```diff
@@ -257,7 +257,7 @@ typedef enum {
 
     The two transform hooks can ONLY be added as transaction hooks.
 
-    TS_VCONN_PRE_ACCEPT_HOOK - Called before the SSL hand
+    TS_VCONN_START_HOOK - Called before the SSL hand
```

Comment:
A better description is "called just after the connection is created, before other activity such as I/O or TLS handshakes".